Mathlib/SetTheory/Ordinal/Basic.lean

```diff
@@ -369,69 +364,65 @@ def principalSegToType {α β : Ordinal} (h : α < β) :
 @[deprecated principalSegToType (since := "2024-08-26")]
 noncomputable alias principalSegOut := principalSegToType
 
+/-- The order type of an element inside a well order.
+
+This is registered as a principal segment embedding into the ordinals, with top `type r`. -/
+def typein (r : α → α → Prop) [IsWellOrder α r] : @PrincipalSeg α Ordinal.{u} r (· < ·) := by
```

Comment:
I guess you'd prefer to use the new notation here?

Reply:
The new notation is specifically for initial/principal segs between two `<` relations, but here we only have one of them!